clients/drcachesim/common/trace_entry.h

```diff
@@ -163,6 +163,30 @@ typedef enum {
      */
     TRACE_TYPE_INSTR_SYSENTER,
 
+    // Trace entry types for prefetch instructions.
+    // Relative order of these enum values is relevant to the handling of prefetch
+    // instructions.
+    TRACE_TYPE_PREFETCH_READ_L1,    /**< Load prefetch to L1 cache. */
```

Comment:
@derekbruening Do you prefer to qualify this identifier with a _TE suffix denoting 'temporal prefetch'?

Reply:
I would vote for only labeling the non-temporal.

Reply:
Ack. I'll let it be as it is.